tests/integration/operatorhub/cmd_service_test.go

```diff
@@ -27,4 +30,27 @@ var _ = Describe("odo service command tests for OperatorHub", func() {
 			Expect(stdOut).To(ContainSubstring("etcdoperator"))
 		})
 	})
+
+	Context("When creating an operator backed service", func() {
+		It("should be able to create EtcdCluster from its alm example", func() {
+			// First let's grab the etcd operator's name from "odo catalog list services" output
+			operators := helper.CmdShouldPass("odo", "catalog", "list", "services")
+			etcdOperator := regexp.MustCompile(`etcdoperator\.*[a-z][0-9]\.[0-9]\.[0-9]`).FindString(operators)
+
+			helper.CmdShouldPass("odo", "service", "create", etcdOperator, "--crd", "EtcdCluster")
+
+			pods := helper.CmdShouldPass("oc", "get", "pods")
+			// Look for pod with example name because that's the name etcd will give to the pods.
+			etcdPod := regexp.MustCompile(`example-.[a-z0-9]*`).FindString(pods)
+
+			ocArgs := []string{"get", "pods", etcdPod, "-o", "template=\"{{.status.phase}}\""}
```

Comment:
Pass namespace as a argument to avoid active namespace collision in parallel run